CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
+* Device with recipe for 'appear' capability by @kostmo in [#1027](https://github.com/swarm-game/swarm/pull/1027)
+* device for setname by @kostmo in [#1028](https://github.com/swarm-game/swarm/pull/1028)
+
+#### LSP improvements
+* Improve OnHover for Def/Let, App and Const by @xsebek in [#986](https://github.com/swarm-game/swarm/pull/986)
+* Include `SrcLoc` info with variable binding sites by @byorgey in [#993](https://github.com/swarm-game/swarm/pull/993)
+* Annotate ASTs with types at every node by @byorgey in [#991](https://github.com/swarm-game/swarm/pull/991)
+* swarm-lang IDE hover by @kostmo in [#972](https://github.com/swarm-game/swarm/pull/972)
+* warn of unused variables by @kostmo in [#983](https://github.com/swarm-game/swarm/pull/983)
+
+#### Driving mode
+* Add a "driving mode" to the REPL by @kostmo in [#819](https://github.com/swarm-game/swarm/pull/819)
+
+#### Web API
+* Expose REPL history via web interface by @kostmo in [#821](https://github.com/swarm-game/swarm/pull/821)
+
+#### Command line options
+* Add full color mode option by @xsebek in [#851](https://github.com/swarm-game/swarm/pull/851)
+
+#### New scenarios
+* New ranching scenario by @kostmo in [#835](https://github.com/swarm-game/swarm/pull/835)
+* new bind tutorial by @kostmo in [#895](https://github.com/swarm-game/swarm/pull/895)
+* A "bucket brigade" of robots by @kostmo in [#885](https://github.com/swarm-game/swarm/pull/885)
+* Ice Cream Shop by @kostmo in [#1008](https://github.com/swarm-game/swarm/pull/1008)
+* word search by @kostmo in [#999](https://github.com/swarm-game/swarm/pull/999)
+* Wolf, Goat, Cabbage scenario by @kostmo in [#944](https://github.com/swarm-game/swarm/pull/944)
+  * Boolean expressions of objective prerequisites by @kostmo in [#927](https://github.com/swarm-game/swarm/pull/927)
+
+#### Achievements
+* achievements board by @kostmo in [#796](https://github.com/swarm-game/swarm/pull/796)
+* first tutorial completion achievement by @kostmo in [#925](https://github.com/swarm-game/swarm/pull/925)
+* Add achievement for losing by @kostmo in [#945](https://github.com/swarm-game/swarm/pull/945)
+
+#### Debugging
+* Better CESK machine pretty-printing by @byorgey in [#948](https://github.com/swarm-game/swarm/pull/948)
+* Add the full expected location for the data directory by @TristanCacqueray in [#908](https://github.com/swarm-game/swarm/pull/908)
+* Extend logs for warnings by @xsebek in [#982](https://github.com/swarm-game/swarm/pull/982)
+
+#### ???
+* show devices that enable commands by @kostmo in [#899](https://github.com/swarm-game/swarm/pull/899)
+* swarm logo animation by @kostmo in [#1018](https://github.com/swarm-game/swarm/pull/1018)
+* support 'orientationMap' by @kostmo in [#1023](https://github.com/swarm-game/swarm/pull/1023)
+* support custom attributes defined in scenarios by @kostmo in [#1058](https://github.com/swarm-game/swarm/pull/1058)
```

Comment:
@kostmo I did not yet figure out where to sort these, suggestions? :mage:

Reply:
How about a "Scenario authoring" section that includes:
* #927
* #1023
* #1058

a "UI/CLI enhancements" section that includes:
* #899
* #819
* #851?

and put #1018 in the "New scenarios" section.

Reply:
@kostmo thanks, done! :heavy_check_mark: